feat: add meeting notes for dec 2nd meeting (#4)
screwdriver-cd/community: meetings/2019-12/dec-02.md

> # Meeting Notes - 2019-11-18

Reviewer: deja vu

## Attendees

- [Jithin Emmanuel](https://github.com/jithine)
- [Keisuke Kumada](https://github.com/kumada626)
- [Shota Yoshika](https://github.com/s-yoshika)

> - Yahoo Japan Team

Reviewer 2: According to our company guidelines, the company name is written as `Yahoo! JAPAN`. So please rewrite as below.
```suggestion
- Yahoo! JAPAN Team
```

## Agenda

From VMG:

- [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830)
- [Parallel Fork](https://github.com/screwdriver-cd/screwdriver/issues/1710)
- [Build timeout not reliable](https://github.com/screwdriver-cd/screwdriver/issues/1710)
- [Updating dependencies](https://github.com/screwdriver-cd/screwdriver/issues/1722)

> From YJapan:

Reviewer 2: ```suggestion
From Yahoo! JAPAN:
```

- [Template/Command list screen shows wrong VERSION](https://github.com/screwdriver-cd/screwdriver/issues/1723)
- [More details on notification email](https://github.com/screwdriver-cd/screwdriver/issues/1668)
- [SD Local](https://github.com/screwdriver-cd/screwdriver/issues/1853)
- [Duplicate PR comments](https://github.com/screwdriver-cd/screwdriver/issues/1858)
- [DB Size is growing](https://github.com/screwdriver-cd/screwdriver/issues/1865)

## Discussion Notes

For [Template/Command list screen shows wrong VERSION](https://github.com/screwdriver-cd/screwdriver/issues/1723) current solution introduces [a bug](https://github.com/screwdriver-cd/screwdriver/issues/1857) with trusted templates. VMG has pinned `datastore-sequelize` to an older version, until this is fixed.

For [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830), discussed workflow, `executor-k8s` to be worked up on after `exececutor-k8s-vm` flow is addressed.

> For [Performance issue of webhook](https://github.com/screwdriver-cd/screwdriver/issues/1468), [prs](https://github.com/screwdriver-cd/screwdriver/pull/1850) have been merged, YJapan team will get performance numbers next time.

Reviewer 2: ```suggestion
For [Performance issue of webhook](https://github.com/screwdriver-cd/screwdriver/issues/1468), [prs](https://github.com/screwdriver-cd/screwdriver/pull/1850) have been merged, Yahoo! JAPAN team will get performance numbers next time.
```
